README.md

```diff
@@ -1 +1,53 @@
-# angular-strict-di-readme
\ No newline at end of file
+# strictDI mode
+
+## Overview
+
+We've used the magic of dependency injection a lot, but did you know it actually is a massive performance overhead? Let's use strict DI to reduce that overhead.
+
+## Objectives
+
+- Describe strictDI mode and Angular's dependency injection annotations
+- Use strictDI mode
+
+## Dependency Injection
+
+Let's say we've got a controller that injects `$scope`, `$timeout` and `UserService`.
+
+```js
+function SomeController($scope, $timeout, UserService) {
+
+}
+
+angular
+	.module('app')
+	.controller('SomeController', SomeController);
+```
+
+When we plug this controller into Angular, Angular checks what dependencies we wants and creates a `$inject` array on our controller:
+
+```js
+function SomeController($scope, $timeout, UserService) {
+
+}
+
+SomeController.$inject = ['$scope', '$timeout', 'UserService'];
+
+angular
+	.module('app')
+	.controller('SomeController', SomeController);
+```
+
+This is where the overhead comes in!
```

Comment:
You lost me here - why is this extra overhead? Is the issue that Angular is creating this even if we haven't injected any dependencies?